oh_staff_ui/tests.py

```diff
@@ -1145,6 +1156,15 @@ def test_valid_mods_name(self):
             b"<mods:namePart>Joe Bruin</mods:namePart>" in ohmods.serializeDocument()
         )
 
+    def test_valid_mods_subject(self):
+        ohmods = self.get_mods_from_interview_item()
+        self.assertTrue(
+            b'<mods:subject authority="local">' in ohmods.serializeDocument()
+        )
+        self.assertTrue(
+            b"<mods:topic>Sample Subject</mods:topic>" in ohmods.serializeDocument()
+        )
+
```

Comment:
@kjallen Since there's logic in `_populate_subjects()` to exclude specific subjects, please add a test to confirm this works as expected.

Reply:
Added in `tests.py`